packages/shared/src/focus-visible.ts

```diff
@@ -152,5 +229,5 @@ export const FocusVisiblePolyfillMixin = <
         }
     }
 
-    return FocusVisibleCoordinator;
+    return FocusVisibleCoordinator as unknown as FVConstructor<T>;
```

Comment:
I couldn't get the typing to work here so I just forced it, seems like its not straightforward to have the mixin constructor apply new properties to the incoming types prototype.